src/main/java/Mediell/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package Mediell;
+
+import java.time.LocalDate;
+
+/** Represents a Task with a Deadline. */
+public class Deadline extends Task{
+    private LocalDate by;
+
+    public Deadline() {
+        super("");
+    }
+
+    public Deadline(String taskName) {
+        super(taskName.split("/by", 2)[0]);
+        by = LocalDate.parse(taskName.split("/by", 2)[1].strip());
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + "(by: " + by + ")";
+    }
+
+    /**
+     * Checks if the provided string is a valid Deadline.
+     * @param format String
+     * @return true if it is a deadline else false
+     */
+    public static boolean isDeadlineFormat(String format) {
+        return format.startsWith("D");
```

Comment:
Should methods be named as a nouns? you might be confusing the boolean convention for variables and methods! I've noticed this in a few of your other methods too.